src/library_egl.js

```diff
@@ -69,7 +69,7 @@ var LibraryEGL = {
           } else if (param == 0x3038 /*EGL_NONE*/) {
               break;
           }
-          attribList += 8;
+          attribList += 8n;
```

Comment:
In general the way we avoid actually having to deal with BigInts in the JS libraries is to use `__sig` specifiers on the functions that receive the pointers.   We use the `p` specifier within the signature to signify that a paramater or return type is a pointer or size_t value.  For example a `__sig` of `vpip` would mean a void function that take a pointer, and integer, and another pointer.  Then the library code will automatically convert pointers to JS numbers (int53 numbers to be precise) at the boundary.

Reply:
Ah, right, i think i did change the __sig of some other method in order to get this to work. I think there are a few places in the opengl code which use "i" instead of "p".